backend/service/service_gov_params.go

```diff
@@ -225,7 +225,7 @@ func (gov GovParamsService) UpdateCurrentValueByKey(kv map[string]interface{}) e
 		}
 		sel := bson.M{document.GovParamsFieldKey: k}
 		update := bson.M{"$set": bson.M{document.GovParamsFieldCurrentValue: vStr}}
-		bulk.Upsert(sel, update)
+		bulk.Update(sel, update, true)
```

Comment:
what is `true` mean in `bulk.Update(sel, update, true)`?

Reply:
```go
// Update queues up the provided pairs of updating instructions.
// The first element of each pair selects which documents must be
// updated, and the second element defines how to update it.
// Each pair matches exactly one document for updating at most.
func (b *Bulk) Update(pairs ...interface{}) {
	if len(pairs)%2 != 0 {
		panic("Bulk.Update requires an even number of parameters")
	}
       // ....
}
```

Reply:
For the lower interface, as usual, we implement it at the beginning. If it is not implemented at the beginning, writing top logics is with much trouble while trying the lower interface. What's more, the costed time is hard to be estimated.